pyccel/ast/variable.py

```diff
@@ -56,6 +56,11 @@ class Variable(TypedAstNode):
         'stack' if memory should be allocated on the stack, represents stack arrays and scalars.
         'alias' if object allows access to memory stored in another variable.
 
+    memory_location: str, default: 'host'
+        'host' the variable can only be accessed by the CPU.
+        'device' the variable can only be accessed by the GPU.
+        'managed' the variable can be accessed by CPU and GPU and is being managed by the Cuda API (memory transfer is being done implicitly).
```

Comment:
This is not necessary. This information is already available in the class_type

Reply:
b414d62